### PR TITLE
use correct revenue field keys

### PR DESCRIPTION
**Pull request by Adam-W1**
https://dluhcdigital.atlassian.net/browse/FS-3661

### CODE REVIEW

File: config/mappings/assessment_mapping_fund_round.py [hunk 1]
```diff
@@ -127,7 +127,7 @@
         "location": "rmBPvK",
         "asset_type": None,
         "funding_one": None,
-        "funding_two": ["qwktzL", "JizgZP"],  # only revenue funding for cyp
+        "funding_two": ["JXKUcj", "OnPeeS"],  # only revenue funding for cyp
```
Comment (RamuniN) [inline config/mappings/assessment_mapping_fund_round.py:130]: you may need to remove this `"field_type": "multiInputField",` from CYP as it's not multiInputField anymore

Comment (Adam-W1) [reply config/mappings/assessment_mapping_fund_round.py:130]: good catch - perhaps we should specify on that field key, something like 'funding_field_type''

**Review by tferns — APPROVED** [review]
lgtm!

File: scripts/data_updates/FS-3661-total-funding-requested-cyp.py
[the remark this answers is not in the document]
Comment (Adam-W1) [reply scripts/data_updates/FS-3661-total-funding-requested-cyp.py:1]: yeah thats right, just once. you should be able to try it locally and see the values update: 
docker exec -it {container id} python -m scripts.data_updates.FS-3661-total-funding-requested-cyp

**Review by RamuniN — APPROVED** [review]

**Pull request merged**